packages/react-examples/.storybook/manager.js

```diff
@@ -1,8 +0,0 @@
-import { addons } from '@storybook/addons';
-import fluentuiTheme from './fluentuiTheme';
-
-addons.setConfig({
```

Comment:
Based on history we should actually keep this file, just remove the `fluentuiTheme` parts

Reply:
updated thx!